NSS-Wacky-Eels/BangazonSite
# Pg ticket7

## Link to Ticket
Closes # [Link](https://github.com/NSS-Enthusiastic-Camels/BangazonAPI/issues/7)

## Description of Proposed Changes
- Created model for OrderProduct
- Created mvc for OrderProduct

## Steps to Test

Outline the steps to test

```sh
git fetch --all
git checkout <pg-ticket7>
user can go to the cart and see the list of product
user can delete the product from the cart
```


## Impacted Areas in Application

List general components of the application that this PR will affect:

* OrderPrductModel and OrderProductController

## Mentions @username

@alexfont321
@Akeen11 
@KaylaReid
@thatmikeparrish

## Definition of Done

1. The project must be fully documented. This includes the following:
    1. Complete README that documents the steps to install the code, how to install any dependencies, or system configuration needed.
    1. Every class must be documented with purpose, author, and methods.
    1. Every method must be documented with purpose and argument list - which itself must contain a short purpose for each argument.
1. The project must be able to run fully, and without errors, on each teammate's system.
1. Fulfills every requirement.
Every line of code has been peer reviewed.
1. For projects that require unit testing, core functionality must be identified and have at least one test for each.
